specification/specs/common.ts

```diff
@@ -13,27 +13,31 @@ type TimeSpan = string
 interface SourceDocument {}
 @class_serializer("ErrorCauseFormatter")
 class ErrorCause {
-  additional_properties: Dictionary<string, any>;
+  shard: Union<integer, string>;
```

Comment:
`integer | string` is translated in the model to `Union<integer, string>`. So should we remove `Union<T, U>` from the spec and unify things with TypeScript union? 

(we could also use the same rule for `Array` vs `[]` although `Array` is an actual TS type)

Reply:
Aye `Union` and `Array` should go now that we support `|` and `[]` literals. 

arrays within unions still error though e.g `string[] | integer`. Will open a ticket to track this after merge

Reply:
I think we could keep `Array`, it's actually more readable in case of unions:
```
Array<string | number>
(string | number)[]
```